src/ipc/ipc4/handler.c

```diff
@@ -277,6 +282,8 @@ int set_pipeline_state(struct ipc_comp_dev *ppl_icd, uint32_t cmd,
 			return 0;
 		case COMP_STATE_ACTIVE:
 		case COMP_STATE_PAUSED:
+			tr_dbg(&ipc_tr, "ipc4: set pipeline %d new cmd state %x: pause from reset",
```

Comment:
I would write active/pause from reset since there's a fallback case, but that's a nit-pick